prov/efa/src/rxr/rxr_init.c

```diff
@@ -704,6 +728,9 @@ EFA_INI
 	if (!lower_efa_prov)
 		return NULL;
 
+	if (rxr_env.enable_shm_transfer && rxr_shm_space_check())
```

Comment:
If we are keeping the name ```rxr_shm_space_check()```, I would prefer 
```
err = rxr_shm_space_check();
if (err)
    rxr_env.enable_shm_transfer = 0;
``` 
which is easier to understand.

Reply:
Done.